extensions/AzureCosmosDBMongoVCore/AzureCosmosDBMongoVCore.FunctionalTests/appsettings.json

```diff
@@ -0,0 +1,73 @@
+{
+    "ServiceAuthorization": {
+        "Endpoint": "http://127.0.0.1:9001/",
+        "AccessKey": ""
+    },
+    "Logging": {
+        "LogLevel": {
+            "Default": "Trace",
+            "Microsoft.AspNetCore": "Trace"
+        }
+    },
+    "KernelMemory": {
+        "Services": {
+            "AzureCosmosDBMongoVCore": {
```

Comment:
Can we add a config for AppName and a default appName please?

Reply:
Added AppName.